CMakeLists.txt

```diff
@@ -97,8 +97,16 @@ include_directories(BEFORE
 add_subdirectory(include/flang)
 add_subdirectory(lib)
 add_subdirectory(runtime)
+add_subdirectory(test)
 add_subdirectory(tools)
 
 configure_file(
   ${FLANG_SOURCE_DIR}/include/flang/Config/config.h.cmake
   ${FLANG_BINARY_DIR}/include/flang/Config/config.h)
+
+enable_testing()
+add_test(NAME Leadz COMMAND leading-zero-bit-count-test)
+add_test(NAME PopPar COMMAND bit-population-count-test)
+add_test(NAME Integer COMMAND integer-test)
+add_test(NAME Logical COMMAND logical-test)
+add_test(NAME Real COMMAND real-test)
```

Comment:
Can the `add_test` commands go in `test/evaluate/CMakeLists.txt`? It seems like the more natural place for them.

Reply:
It seems to work if `enable_testing()` is moved before `add_subdirectory`.